src/MartinCostello.BrowserStack.Automate/AutomatePlanStatus.cs

```diff
@@ -38,7 +38,7 @@ public class AutomatePlanStatus
         /// <summary>
         /// Gets or sets the number of parallel sessions currently queued.
         /// </summary>
-        [JsonProperty("queued_sessions")]
+        [JsonProperty("queued_sessions", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
This is currently returning 0, but I'm preemptively handling the null to avoid any surprises in the future...